lava-v2-jobs-from-api.py

```diff
@@ -197,20 +195,19 @@ def jinja_render(job):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument("--token", help="KernelCI API Token")
+    parser.add_argument("--token", help="KernelCI API Token", required=True)
```

Comment:
This should not be required on the command line as the token can be read from the config file (and even from the environment, looking at get_config()).  I think there should rather be a test in the code to check it's not empty after calling config.get('token').